### ISSUE TITLE
1.0.3 doesn't update location just displays "in main menu" all the time

### ISSUE THREAD

**Issue by mondsucht** [amitojsingh366/Amitojs-Minecraft-RPC#2]
After building and running the latest version, I have noticed location and held items are no longer shown. Instead in discord "in main menu" is always visible. No other issues appeared. Release 1.0.2 downloaded from here does not have this problem.

Screenshots

- In [1.0.3](https://i.imgur.com/Xfh2zpf.png)

- In [1.0.2](https://i.imgur.com/Toj2CN8.png)

Both times running minecraft 1.16.5 on jre-openjdk-headless 15.0.2.u7 and Forge 36.1.18 with no other mods installed.

AmitojsMinecraftRPC-1.0.3-shaded.jar was built on Arch Linux using jdk8-openjdk 8.u292

I can provide logs if needed.

**Comment by amitojsingh366** [comment]
yes im aware of this, i have actually dropped development on this mod and now use [this](https://github.com/amitojsingh366/Amitojs-Minecraft-RPC-Fabric) *(it's the same mod but for fabric)*. the last confirmed working version for this mod is [1.0.2](https://github.com/amitojsingh366/Amitojs-Minecraft-RPC/releases/tag/v1.0.2)
